projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -52,7 +55,7 @@ export function updateVDKMenu(commands: CommandRegistry) {
                 body: <DeleteJobDialog jobName={defaultJobName} jobTeam={sessionStorage.getItem("delete-job-team")!}></DeleteJobDialog>,
                 buttons: [Dialog.okButton(), Dialog.cancelButton()],
             }).then(async result => {
-                if (!result.value) {
+                if (result.button.accept) {
                     let bodyMessage = 'Do you really want to delete the job with name ' + sessionStorage.getItem("delete-job-name") + " from " + sessionStorage.getItem("delete-job-rest-api-url") + "?"
                     showDialog({
```

Comment:
I find it every odd that you mix async/awaits with then/catch. 
I think you should try use async/await everywhere.